dynamic_rest/viewsets.py

```diff
@@ -337,4 +337,91 @@ def list_related(self, request, pk=None, field_name=None):
 
 
 class DynamicModelViewSet(WithDynamicViewSetMixin, viewsets.ModelViewSet):
-    pass
+
+    ENABLE_BULK_PARTIAL_CREATION = (
+        dynamic_settings.get('ENABLE_BULK_PARTIAL_CREATION', False))
```

Comment:
The setting that you added to `dynamic_rest.conf` won't be reflected here unless you use the new form for settings:

```
from dynamic_rest.conf import settings
...
ENABLE_BULK_PARTIAL_CREATION = settings.ENABLE_BULK_PARTIAL_CREATION
```

There also seems to be a mismatch of defaults; you have default of False here, but default of True in `conf.py`.